modular_skyrat/master_files/code/game/atoms.dm

```diff
@@ -0,0 +1,8 @@
+/// Stop All Animations nulls the mob's transform, so we have to call update_body_size to ensure that it gets scaled properly again
```

Comment:
Shouldnt be a code doc so it doesn't override any potential doc of this proc from tg